<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueForm</class>
 <widget class="QDialog" name="IssueForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Submit An Issue</string>
  </property>
  <widget class="QLabel" name="lblSummary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>51</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Summary:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSummary">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboAsset">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>411</width>
     <height>24</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblAsset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Asset:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPriority">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboPriority">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>95</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDetail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Describe the issue you are encountering in detail:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtDetail">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>501</width>
     <height>241</height>
    </rect>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEmail">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblEmail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
